Files/UserControls/NavigationToolbar.xaml

```diff
@@ -834,89 +812,37 @@
-                                            Stretch="Uniform" />
-                                    </Grid>
-                                    <FontIcon
-                                        x:Name="EmptyIconGlyph"
-                                        x:Load="{x:Bind LoadUnknownTypeGlyph}"
-                                        Glyph="&#xE7C3;" />
-                                    <FontIcon
-                                        x:Name="FolderGlyphIcon"
-                                        Width="24"
-                                        Height="24"
-                                        HorizontalAlignment="Stretch"
-                                        VerticalAlignment="Stretch"
-                                        x:Load="{x:Bind LoadFolderGlyph, Mode=OneWay}"
-                                        Glyph="&#xF12B;" />
-                                </Grid>
-                                <TextBlock VerticalAlignment="Center" Text="{x:Bind ItemName}" />
-                            </StackPanel>
-                        </Grid>
-                    </DataTemplate>
-                </AutoSuggestBox.ItemTemplate>
-                <AutoSuggestBox.KeyboardAccelerators>
-                    <KeyboardAccelerator
-                        Key="Escape"
-                        Invoked="SearchBox_Escaped"
-                        Modifiers="None" />
-                </AutoSuggestBox.KeyboardAccelerators>
-            </AutoSuggestBox>
+                Visibility="{x:Bind IsSearchBoxVisible, Mode=OneWay}">
+                <uc:SearchBox.KeyboardAccelerators>
+                    <KeyboardAccelerator Key="Escape" Invoked="SearchBox_Escaped" />
+                </uc:SearchBox.KeyboardAccelerators>
+            </uc:SearchBox>
 
             <Button
                 x:Name="SearchButton"
                 x:Uid="NavSearchButton"
                 Grid.Column="2"
-                Width="36"
-                Height="32"
-                AccessKey="I"
-                AutomationProperties.Name="Search"
+                Width="36" Height="32"
                 Background="Transparent"
+                Style="{StaticResource ToolBarButtonStyle}"
+                AccessKey="I"
                 Click="SearchButton_Click"
                 IsEnabled="{x:Bind IsPageTypeNotHome, Mode=OneWay}"
-                Style="{StaticResource ToolBarButtonStyle}"
+                AutomationProperties.Name="Search"
                 ToolTipService.ToolTip="Search (Ctrl+F)">
                 <FontIcon FontSize="14" Glyph="{x:Bind SearchButtonGlyph, Mode=OneWay}" />
                 <Button.KeyboardAccelerators>
-                    <KeyboardAccelerator
-                        Key="F"
-                        IsEnabled="False"
-                        Modifiers="Control" />
+                    <KeyboardAccelerator Key="F" Modifiers="Control" IsEnabled="False" />
```

Comment:
Can you revert the formatting changes made to this button?

Reply:
@yaichenbaum Ok, I'll do this.

Reply:
I fixed it from my end 🙂